File: sub-model/src/main/java/org/github/base/entity/SysRoleDeptEntity.java
package org.github.base.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.IdType;
import org.github.base.AbstractEntity;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.*;

/**
 * <p>
 * 角色和部门关联表
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-21
 */
@Setter
@Getter
@EqualsAndHashCode(callSuper = true)
@TableName("sys_role_dept")
public class SysRoleDeptEntity extends AbstractEntity {
    /** UID */
    private static final long serialVersionUID = 1L;

    /** 角色ID */
    @TableId(value = "role_id", type = IdType.UUID)
    private Long roleId;

    /** 部门ID */
    @TableField("dept_id")
    private Long deptId;
}

File: sub-model/src/main/java/org/github/base/entity/SysRoleDeptEntity.kt
package org.github.base.entity

import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.AbstractEntity

/**
 * <p>
 * 角色和部门关联表
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@TableName("sys_role_dept")
class SysRoleDeptEntity: AbstractEntity() {

    /** 角色ID */
    var roleId: Long? = null

    /** 部门ID */
    var deptId: Long? = null

}
